### ISSUE TITLE
Strip feeder setup: Add diagram showing ref hole and first part

### ISSUE THREAD

**Issue by FlyingLotus1983** [TinWhiskers/openpnp#3]

**Comment by FlyingLotus1983** [comment]
@vonnieda where would the diagram showing the ref hole and first part be?  Is this in the simulated feeders on the camera view?  Or is this more something that goes in the wizard tab?

**Comment by vonnieda** [comment]
Goes in the wizard, but only if we're not doing more advanced vision based setup. Basically, all of these cut strip setup issues depend on each other and some cancel the others out. If we do vision based setup like in #7 we don't need this at all. It would probably be better to just track it in a single issue and decide what we're actually building before we run off in different directions.

**Comment by FlyingLotus1983** [comment]
Makes sense, I'll leave them all open for the moment, as we try stuff and experiment, we can hopefully find a winning combination to use, and close the ones that don't pan out.
